hack/jenkins/job-configs/kubernetes-e2e.yaml

```diff
@@ -5,7 +5,8 @@
         daysToKeep: 7
     builders:
         - shell: |
-            curl -fsS --retry 3  "https://raw.githubusercontent.com/kubernetes/kubernetes/{branch}/hack/jenkins/e2e.sh" | bash -
+            curl -fsS --retry 3  "https://raw.githubusercontent.com/kubernetes/kubernetes/{branch}/hack/jenkins/e2e-jobs.sh" | bash -
+            curl -fsS --retry 3  "https://raw.githubusercontent.com/kubernetes/kubernetes/{branch}/hack/jenkins/e2e-runner.sh" | bash -
```

Comment:
I think we want to use `master` here, so we don't need to update the runner in 3 places?